demo/eff.css

```diff
@@ -0,0 +1,93 @@
+html, textarea, input {
```

Comment:
Overall, it seems that tabs are used for indention. As long as you don't have strong preferences and reasons for using tabs in .css files, would you mind using spaces for indention in .css files as well to be consistent within the project?

The `demo/eff.js` file also uses tabs instead of spaces for indention.